tiledb/sm/c_api/tiledb.h

```diff
@@ -915,6 +915,11 @@ TILEDB_EXPORT void tiledb_config_free(tiledb_config_t** config);
  *    If `true`, an error will be thrown if there are cells with coordinates
  *    lying outside the domain during sparse fragment writes.  <br>
  *    **Default**: true
+ *    `sm.read_range_oob_error` <br>
+ *    If `true`, this will check ranges for read with out-of-bounds on the
```

Comment:
let's use the word "error" somewhere in this first sentence so that its slightly more obvious that `true`==error and `false`==warning.

Reply:
@joe-maley what if switch to a `sm.read_range_oob_check` with values of `error` or `warn`? Or do you only want the description adjusted?

Reply:
I think that sounds better if you don't mind making the change.